amazon-location-route-planner/vite.config.js

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "vite";
+import vue from "@vitejs/plugin-vue";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [vue()],
+  resolve: {
+    alias: {
+      "./runtimeConfig": "./runtimeConfig.browser",
+    },
+  },
```

Comment:
As pointed out in another comment by @zachelliottwx I'd recommend copying over the port/server config from one of the existing samples.

The port number is irrelevant (albeit handy for us in terms of consistency), but the vite config needs to be instructed to listen for connections from different hosts/addresses. This is needed for Cloud9 as the app is served via reverse proxy once previewed in the browser

Reply:
fixed in [aee5817](https://github.com/aws-samples/amazon-location-samples/pull/247/commits/aee5817296be0322c99adba4deb933c638e358b2) and [4563ff3](https://github.com/aws-samples/amazon-location-samples/pull/247/commits/4563ff365d934b4b597563b78af17919e954a2c3).